src/test/resources/application.yml

```diff
@@ -0,0 +1,14 @@
+spring:
+  datasource:
+    url: jdbc:postgresql://${AQTS_DATABASE_ADDRESS}:${AQTS_DATABASE_PORT}/${AQTS_DATABASE_NAME}?currentSchema=${AQTS_SCHEMA_NAME}
+    username: ${AQTS_SCHEMA_OWNER_USERNAME}
+    password: ${AQTS_SCHEMA_OWNER_PASSWORD}
+    hikari:
+      maximum-pool-size: 2
+
+maxTimeSeriesCorrectedData: 10
```

Comment:
This property might be related to the MAX_TIME_SERIES_CORRECTED_DATA property in the serverless.yml file, so my previous comment about the serverless line not being needed might be false...